test/test/helpers/immutable-chai/index.js

```diff
@@ -0,0 +1,16 @@
+import immutable from 'immutable';
+
+export default function useImmutable( _chai ) {
+	var Assertion = _chai.Assertion;
+
+	Assertion.addMethod( 'immutablyEqual', function( other ) {
+		this.assert(
+			immutable.is( this._obj, other ),
+			'Expected #{exp} to pass Immutable.is against #{act}',
```

Comment:
I'm not native speaker and nitpicky, but would it make sense to have `against #{act}` in both messages?